econml/solutions/causal_analysis/_causal_analysis.py

```diff
@@ -537,6 +554,9 @@ def process_feature(name, feat_ind):
                     _CausalInsightsConstants.CategoricalColumnKey: [name],
                     _CausalInsightsConstants.EngineeredNameKey: [name]
                 }
+                # calculate a "typical" treatment value, using the mean of the absolute value of non-zero treatments
+                treatment_value = np.mean(np.abs(T[T != 0]))
```

Comment:
Yes: if for some units it's positive and some it's negative, we don't want to average it to zero: the "typical" treatment value is the magnitude, and when we learn a policy we'll either increase or decrease by some fraction of the typical (currently 10%)

Reply:
You're right that if the treatment is never positive, then having the "typical" treatment also be negative might be more natural in some sense, but since our policy is expressed in terms of "increase" vs. "decrease", we want to be multiplying our policy values by a positive number to maintain the correct sign.